lib/processors/bundlers/moduleBundler.js

```diff
@@ -33,10 +33,12 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  *      the ui5loader is available.
  *	</li>
  *	<li>
- *		<code>require</code>: A 'require' section is transformed into a sequence of jQuery.sap.require calls. The
+ *		<code>require</code>: A 'require' section is transformed into a sequence of 'sap.ui.require' calls. The
  *		list will be resolved like an include pattern list in any of the other sections and for each of the resolved
- *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.requireSync' is
- *		used instead.
+ *		modules, a 'sap.ui.require' will be created.
```

Comment:
The description still does not fit to the actual behavior. Please check for example: https://github.com/SAP/ui5-builder/blob/7691b08c4cd8abc7fe3922ec913f80dca372ba4f/lib/lbt/bundle/Builder.js#L80-L85

There is only one sap.ui.require call per section, and the modules are loaded in parallel. Only when using `async: false`, they are loaded with individual `sap.ui.requireSync` calls in sequence.

Reply:
What about now?
Is it better?

Reply:
Yes, looks better 👍🏻